tools/dev/msftidy.rb

```diff
@@ -12,16 +12,6 @@
 
 CHECK_OLD_RUBIES = !!ENV['MSF_CHECK_OLD_RUBIES']
 SUPPRESS_INFO_MESSAGES = !!ENV['MSF_SUPPRESS_INFO_MESSAGES']
-TITLE_WHITELIST = %w{
```

Comment:
This wretched thing has been unmaintainable. We'll continue to spot check during review.